### PR TITLE
Menu with material design lite

### PR DESCRIPTION
**Pull request by jailby**

### CODE REVIEW

**Comment by barbeau** [comment]
@jailby Can you also please post a screenshot of what it looks like?

**Comment by jailby** [comment]
The following is the screenshot for the credit:
image

This other screenshot is for the separator:
image

**Comment by barbeau** [comment]
Visually looks good to me.  @jmfield2 will need to review code.

File: src/client/js/otp/config.js [hunk 1]
```diff
@@ -174,7 +174,8 @@ otp.config = {
     	title: 'Icon Legend',
     	content: '<p><img src="images/locationSpot.svg" height="15" width="15"> : Current Location<br>\
     		<img src="images/busStopButton.png" height="15" width="15"> : BullRunner Bus Stop<br>\
-    		<img src="images/stop20.png" height="15" width="15"> : HART Bus Stop<br>'      	
+    		<img src="images/stop20.png" height="15" width="15"> : HART Bus Stop<br> \
+    		<p> We would like to acknowledge the support and funding assistance provided by the USF Student Green Energy Fund, Center for Urban Transportation Research, and Florida Department of Transportation.'
```
Comment (jmfield2) [inline src/client/js/otp/config.js]: It looks like this is missing two closing `</p>` tags (before the ')  Otherwise, seems to be good

**Comment by jailby** [comment]
@barbeau the `</p>` tags have been added. It should be good to merge now.

**Comment by barbeau** [comment]
@jailby Ok - can you post new screenshot?

@jmfield2 Can you please review code and merge if it looks good?

**Comment by jailby** [comment]
It did not change anything but the following is a screenshot of the current look of this widget:
image

**Comment by jailby** [comment]
The coloring of the button corresponding to the widget open is done.

The following are screenshots of this changes:

When open the website (same thing than open with #tripplanner)
image

When open the website with #layers: 
image

When open the website with #map: 
image

I did not include screenshot for when switching between widgets because it is the same images.

**Comment by jmfield2** [comment]
Cool! What happens if you just click the widget titlebar from the bottom
when minimized?

**Comment by jailby** [comment]
When you manually (without clicking on a button from the menu) minimize or maximize the corresponding button are not being colored.

**Comment by jmfield2** [comment]
OK - I see why you wanted to know how to change the class inside of minimize/unminimize  lol.. 

Did you mean for the class to be "slected_item" and not "selected_item" ?  Otherwise, I think this looks good.

**Comment by jailby** [comment]
Yes I meant "selected_item". Dammit dyslexia go away lol.

My last commit reflect this change.

@barbeau can you make sure that this is the behavior you wanted? Manually minimizing/maximizing a widget does not color the corresponding menu's button.

**Comment by jmfield2** [comment]
Probably the easiest way to adjust for the manual interactions is to add
the add and removeclass calls inside of js/otp/widgets/Widget.js minimize
and unminimize

But hopefully it is easy to determine which widget is being targeted (
this.id maybe) to determine the correct target menu list id..

**Comment by jmfield2** [comment]
I forgot one thing - an issue might arise where a widget was below another
and then minimized...or other cases.. So, we need to actually determine
which widget will be in front of all others after performing the action -
there are zindex methods that might help with that...

**Comment by jailby** [comment]
I think we should talk during the meeting of what is the best/wanted behavior because from what Sean told me if both are open both should be colored (no matter which one is at the top).

If we also add/remove the coloring in the function minimize/unminimize we need to remove it from the onClick tag otherwise it will conflict.

**Comment by barbeau** [comment]
I think the current behavior that @jailby has is ok - startup behavior is correct, and after that only clicks on the menu change the color.  The manual minimize/unminimize of windows shouldn't change anything.

**Comment by jailby** [comment]
@jmfield2, since @barbeau has confirmed in the previous comment that this is the behavior wanted, can you check quickly the code?

Once it is done I will merge this PR and close #132.

**Comment by jmfield2** [comment]
Looks good!

**Pull request merged**